reference/api/indexes.md

```diff
@@ -214,3 +214,51 @@ Delete an index.
 ```
 
 You can use the response's `taskUid` to [track the status of your request](/reference/api/tasks.md#get-one-task).
+
+## Swap indexes
+
+<RouteHighlighter method="POST" route="/indexes/swap-indexes"/>
+
+Swap the documents, settings, and task history of two or more indexes. **You can only swap indexes in pairs.** However, a single request can swap as many index pairs as you wish.
+
+Swapping indexes is an atomic transaction: **either all indexes are successfully swapped, or none are.**
+
+Swapping `indexA` and `indexB` will also replace every mention of `indexA` by `indexB` and vice-versa in the task history. `enqueued` tasks are left unmodified.
+
+[To learn more about index swapping, refer to this short guide.](/learn/core_concepts/indexes.md#swapping-indexes)
+
+### Body
+
+An object containing some string arrays, each called `indexes`.
+
+| Name           | Type             | Default value | Description                                |
+| :------------- | :--------------- | :------------ | :----------------------------------------- |
+| **`indexes`*** | Array of strings | N/A           | Array of the two `indexUid`s to be swapped |
+
+You can send multiple `indexes` arrays in a single request. Each `indexes` array should contain only two elements: the `indexUid`s of the two indexes to be swapped.
```

Comment:
```suggestion
You can send multiple `indexes` arrays in a single request. Each `indexes` array should contain only two elements: the `indexUid`s of the two indexes to be swapped.
```

I would have put that information in a visible insert like `::: note` as a quick-tip; it's a gold nugget info to have.